### PR TITLE
Create CCC-TAC-Minutes-2023-04-06.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  ***ACTION*** Helen to ask about turning on the comment feature on the website for blog comments
+
+**2023 Goals**
+
+* DM addressed 2023 [plans](https://docs.google.com/document/d/1BLsI0hv9ybHl-FBNqHp6bJzy6ng8yKs__556bTqBswc/edit)
+* Project Progression
+  * Howard Huang (HH) discussed the [PR #160](https://github.com/confidential-computing/governance/pull/160) for refining the definition of the project progression policy
+    * Need to clarify and seperate the benefits for each state
+    * DT proposed that the TAC remove the 2 later unapproved states originally suggested but not approved
+    * Headed home suggested that there be more benefits for the sandbox stage
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md:108]: "Headed home"???

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+* HH mentioned a new project [Bao](https://github.com/bao-project) and asked the TSC chair to speak at a future TAC meeting
+
+
+**TAC Tech Talk**
+* Towards Trust Minimization in TEEs and Applications - Fan Zhang (CDCC)
+
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md:114]: Was there no discussion worth noting?

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md (hunk 1 above)
Comment (krtaylor) [reply TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md]: the downside of autocorrect shortcuts

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md (hunk 2 above)
Comment (dcmiddle) [reply TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md:114]: I think this is fine given the work is not published yet. The community will be able to read the publication soon enough.

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  * Howard Huang (HH) discussed the [PR #160](https://github.com/confidential-computing/governance/pull/160) for refining the definition of the project progression policy
+    * Need to clarify and seperate the benefits for each state
+    * DT proposed that the TAC remove the 2 later unapproved states originally suggested but not approved
+    * HH suggested that there be more benefits for the sandbox stage
+* HH mentioned a new project [Bao](https://github.com/bao-project) and asked the TSC chair to speak at a future TAC meeting
+
+
+**TAC Tech Talk**
+* Towards Trust Minimization in TEEs and Applications - Fan Zhang (CDCC)
+
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md:114]:
```suggestion
Though the recording is available, as the work is not yet published, we did not record slides or take minutes of discussion items.
No directional or decision impacting items were discussed.
```

File: TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    * [Cloud Controls Matrix](https://cloudsecurityalliance.org/research/cloud-controls-matrix/)
+  * Governance research - draft document on attestation service governance being developed
+  *Q2 plans
+    * CSA - publish a position paper, start research into updating CCM
+    * NIST - submit comments on NISTIR-8320D (hardware-based confidential computing), searching for contacts for help
+    * Governance research - complete attestation service governance document, research CI/CD integration
+* DT asked about conformance plans, MN and AF agreed to address conformance and certification in upcoming meetings in Q2
+
+**Project Website funding**
+* KT presented the website updates and funding
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md:130]:
```suggestion
* KT presented the Gramine website updates and funding 
```

**Pull request merged**